### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by tatiNo5**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.0.0
+0.0.1
```
Comment (Foriger) [inline VERSION]: It is not a minor change IMO.

File: schema.go [hunk 2]
```diff
@@ -400,4 +435,327 @@ var metaSchemaData = []byte(`
         "exclusiveMinimum": [ "minimum" ]
     },
     "default": {}
-}`)
+}`),
+	draft06: []byte(`{
+		"$schema": "http://json-schema.org/draft-06/schema#",
```
Comment (Foriger) [inline schema.go]: Schemas could be in files. Why there are such hardcoded strings?

**Comment by Foriger** [comment]
Why package manager dependencies are committed in source code? The general rule in most programming languages is to not commit build artifacts.

File: VERSION (hunk 1 above)
Comment (tatiNo5) [reply VERSION]: make sense.. changing it

File: schema.go (hunk 2 above)
Comment (tatiNo5) [reply schema.go]: That map is used only in that file and is not supposed to be used of any other which is why it here

**Pull request merged**